### ISSUE TITLE
Unable to execute Sonar - The rule 'XYZ' does not exist

### ISSUE THREAD

**Issue by giacgbj** [Backelite/sonar-swift#2]
When the command sonar-runner is executed, the result is: 
```
18:02:21.119 INFO  - Processing SwiftLint report sonar-reports/projectName-swiftlint.txt
INFO: ------------------------------------------------------------------------
INFO: EXECUTION FAILURE
INFO: ------------------------------------------------------------------------
Total time: 5.209s
Final Memory: 12M/290M
INFO: ------------------------------------------------------------------------
ERROR: Error during Sonar runner execution
org.sonar.runner.impl.RunnerException: Unable to execute Sonar
	at org.sonar.runner.impl.BatchLauncher$1.delegateExecution(BatchLauncher.java:91)
	at org.sonar.runner.impl.BatchLauncher$1.run(BatchLauncher.java:75)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.sonar.runner.impl.BatchLauncher.doExecute(BatchLauncher.java:69)
	at org.sonar.runner.impl.BatchLauncher.execute(BatchLauncher.java:50)
	at org.sonar.runner.api.EmbeddedRunner.doExecute(EmbeddedRunner.java:102)
	at org.sonar.runner.api.Runner.execute(Runner.java:100)
	at org.sonar.runner.Main.executeTask(Main.java:70)
	at org.sonar.runner.Main.execute(Main.java:59)
	at org.sonar.runner.Main.main(Main.java:53)
Caused by: The rule 'SwiftLint:trailing_semicolon' does not exist.
```

I have the same result using the default "Sonar way" profile (which has only a rule enabled) and a custom profile I created (which has all the 25 rules that exist for Swift).

**Comment by zippy1978** [comment]
Hi,

SwiftLint:trailing_semicolon is missing (version 0.1.0 was made with SwiftLint 0.3.0).
I will issue a new release soon with SwiftLint 0.4.0 new rules.

**Comment by zippy1978** [comment]
Fixed on develop branch

**Comment by giacgbj** [comment]
I confirm it works on the develop branch.

**Comment by zippy1978** [comment]
Now available in release 0.1.1
